setup.cfg

```diff
@@ -89,7 +89,7 @@ test =
 doc =
   sphinx!=1.2.0,!=1.3b1,>=1.1.2
   sphinx_rtd_theme
-  openstack-doc-tools>=1.0.1
+  cliff>=2.10
```

Comment:
Cliff is in requirements.txt so no need to list it here. Just update the version if it's needed in `requirements.txt`.